src/encoded/schemas/experiment_seq.json

```diff
@@ -122,6 +122,13 @@
             "type": "number",
             "lookup": 113,
             "use_for": ["MARGI"]
+        },
+        "rna_strandness":{
+            "title": "RNA strand-specific direction",
+            "description": "The direction of the RNA strand used for library preparation",
```

Comment:
title: RNA library strandedness
description: For RNA libraries, do reads correspond to the forward RNA strand, the complementary reverse strand, or both (unstranded)